pages/common/docker-start.md

```diff
@@ -0,0 +1,20 @@
+# docker start
+
+> Start one or more stopped containers.
+> More information: <https://docs.docker.com/engine/reference/commandline/start/>.
+
+- Show help:
+
+`docker start`
+
+- Start a docker container:
+
+`docker start {{container}}`
+
+- Start a container, attaching `stdout` and `stderr` and forwarding signals:
+
+`docker start --attach {{container}}`
+
+- Start one or more space-separated containers:
+
+`docker start {{container(s)}}`
```

Comment:
Can we use this format? I am not sure which one is better. But I remember I see the following format somewhere.

```suggestion
`docker start {{container1}} {{container2}}`
```

Reply:
I Looked this `cat * | grep "space-separated" -C3` in `pages/common/` and I found results like:

```
- Disable one or more space-separated modules:

`magento module:disable {{module(s)}}`
```
and
```
- Analyse the specified space-separated directories:

`phpstan analyse {{path/to/directory}}`
```

Reply:
Make sense.